src/ai/backend/manager/models/endpoint.py

```diff
@@ -379,6 +384,9 @@ class Meta:
 
     endpoint_id = graphene.UUID()
     image = graphene.String()
+    image_id = graphene.UUID(description="Added at 23.09.9")
+    image_row = graphene.Field(Image, description="Added at 23.09.9")
```

Comment:
`row` is a very specific noun, representing database record. Since the intension of serving this information is to expose full information of image "object", let's change the naming to do so.
```suggestion
    image = graphene.String()
    image_id = graphene.UUID(description="Added at 23.09.9")
    image_object = graphene.Field(Image, description="Added at 23.09.9")
```